src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -227,15 +231,24 @@ private static String escapeXml(String str) {
 
   private void importResults(SensorContext context) {
     File analysisOutput = new File(toolOutput(context.fileSystem()), ANALYSIS_OUTPUT_XML_NAME);
-
     new AnalysisResultImporter(context, fileLinesContextFactory, noSonarFilter).parse(analysisOutput);
+
+    File highlightInfoFile = new File(toolOutput(context.fileSystem()), HIGHLIGHT_OUTPUT_PROTOBUF_NAME);
+    if (highlightInfoFile.isFile()) {
```

Comment:
Shouldn't you log (at least in debug) when no highlighting file is found? This would help to diagnose why colorization would be missing.